hassio/snapshots/hassio-snapshot.html

```diff
@@ -0,0 +1,260 @@
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../bower_components/paper-dialog/paper-dialog.html'>
+<link rel='import' href='../../bower_components/paper-checkbox/paper-checkbox.html'>
+<link rel='import' href='../../bower_components/paper-button/paper-button.html'>
+
+<dom-module id='hassio-snapshot'>
+  <template>
+    <style include='ha-style'>
+      .content {
+        width: 400px;
+        padding: 16px 48px 0 48px;
+      }
+      iron-icon {
+        float: left;
+        color: var(--primary-color);
+      }
+      paper-checkbox {
+        display: block;
+        margin: 4px 8px 4px 48px;
+      }
+      .snapshot {
+        margin-bottom: 32px;
+        margin-left: 32px;
+        font-size: 1.2em;
+      }
+      .snapshot-type {
+        color: var(--secondary-text-color);
+        font-size: 0.9em;
+      }
+      .addon-version {
+        color: var(--secondary-text-color);
+      }      
+      .card-actions {
+        border-top: 1px solid #e8e8e8;
+        padding: 5px 16px;
+        margin: 0;
+      }
+      .warning {
+        color: var(--google-red-500);
+      }
+      .error {
+        color: var(--google-red-500);
+      }
+      @media screen and (max-width: 600px) {
+        paper-dialog {
+          position: absolute !important;
+          bottom: 0; left: 0; right: 0;
+          border-bottom-left-radius: 0;
+          border-bottom-right-radius: 0;
+          margin: 0;
+        }
+      }
+    </style>
+    <paper-dialog id='dialog' with-backdrop on-iron-overlay-closed='dialogClosed'>
+      <div class='content'>
+        <iron-icon icon='[[computeIcon(snapshot.type)]]' class='primary'
+        ></iron-icon>
+        <div class='snapshot'>
+          [[computeName(snapshot)]]
+          <div class='snapshot-type'>
+            [[computeType(snapshot.type)]] ([[computeSize(snapshot.size)]])<br/>
+            [[formatDatetime(snapshot.date)]]
+          </div>
+        </div>
+        <div>
+          Home Assistant:
+          <paper-checkbox checked='{{restoreHass}}'>
+            Home Assistant [[snapshot.homeassistant]]
+          </paper-checkbox>
+        </div>
+        <div>
+          Folders:
+          <template is='dom-repeat' items='[[snapshot.folders]]'>
+            <paper-checkbox checked='{{item.checked}}'>
+              [[item.name]]
+            </paper-checkbox>
+          </template>
+        </div>
+        <template is='dom-if' if='[[showAddons(snapshot.addons)]]'>
```

Comment:
Since `if` is a truthy check, you can just use the length of the addons array to see if you should show them. The value 0 is considered false, non-zero is true.

```html
<template is='dom-if' if='[[snapshot.addons.length]]'>
```